docs/topics/auth/customizing.txt

```diff
@@ -268,6 +268,14 @@ the check performed). An authentication backend will receive the keyword
 parameters ``obj`` and ``user_obj`` for each object related authorization
 method and can return the object level permission as appropriate.
 
+Note that the default :class:`~django.contrib.auth.backends.ModelBackend`
+always returns False when queried for object level permissions. If you want to
+pass an object but still include all general permissions, you can either adapt
```

Comment:
This advice doesn't provide much clarity to me. I think a solution will be based on what backends a project is using so I'm not sure if there's any general advice to give. In any case, I wouldn't include this commit as part this PR as it's not really related.